config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml

```diff
@@ -0,0 +1,66 @@
+periodics:
+- name: ci-kubernetes-e2e-kubeadm-kind-master
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20190205-d83780367-master
+      env:
+      # skip serial tests and run with --ginkgo-parallel
+      - name: "PARALLEL"
+        value: "true"
+      args:
+      - "--job=$(JOB_NAME)"
+      - "--root=/go/src"
+      - "--repo=k8s.io/kubernetes=master"
+      - "--repo=k8s.io/kubeadm=master"
+      - "--repo=sigs.k8s.io/kind=master"
+      - "--service-account=/etc/service-account/service-account.json"
+      - "--upload=gs://kubernetes-jenkins/logs"
+      - "--scenario=kubernetes_e2e"
+      - "--timeout=30m"
+      - "--"
+      - "--cluster="
+      - "--build=bazel"
+      - "--provider=skeleton"
+      - "--deployment=kind"
+      - "--kind-binary-version=latest"
+      - "--kind-config./../../k8s.io/kubeadm/tests/e2e/kind/single-cp/single-cp.yaml"
+      - "--test-cmd=./../../k8s.io/kubeadm/tests/e2e/kind/test-cmd-basic.sh"
+      # we need privileged mode in order to do docker in docker
+      securityContext:
+        privileged: true
+      # kind needs /lib/modules and cgroups from the host
+      volumeMounts:
+      - mountPath: /lib/modules
+        name: modules
+        readOnly: true
+      - mountPath: /sys/fs/cgroup
+        name: cgroup
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
+          # during the tests more like 3-20m is used
+          cpu: 2000m
+    volumes:
+    - name: modules
+      hostPath:
+        path: /lib/modules
+        type: Directory
+    - name: cgroup
+      hostPath:
+        path: /sys/fs/cgroup
+        type: Directory
+    # trialing this on kind jobs, we are using FQDN for in-cluster services, now
+    # so use ndots 1 to improve dns performance
+    # TODO(bentheelder): consider setting this at the cluster level instead
+    dnsConfig:
+      options:
+        - name: ndots
+          value: "1"
```

Comment:
^ the rest is just a copy-paste. my assumption is that we are going to need it for all jobs?

Reply:
we can ditch the ndots, it makes the config more verbose and results have been inconclusive imho. it's difficult to measure